components/api_request_helper/api_request_helper.h

```diff
@@ -30,32 +31,38 @@ class APIRequestResult {
   APIRequestResult();
   APIRequestResult(int response_code,
                    std::string body,
+                   base::Value value_body,
```

Comment:
can you please write a comment describing all the parameters, esp body/value_body and final_url